plugins/amazonq/codetransform/jetbrains-community/src/software/aws/toolkits/jetbrains/services/codemodernizer/constants/CodeModernizerUIConstants.kt

```diff
@@ -67,13 +81,54 @@ class CodeModernizerUIConstants {
             fill = GridBagConstraints.HORIZONTAL
             anchor = GridBagConstraints.NORTH
         }
+
+        val transformationPlanAppendixConstraint = GridBagConstraints().apply {
+            gridx = 0
+            gridy = GridBagConstraints.RELATIVE
+            fill = GridBagConstraints.HORIZONTAL
+            anchor = GridBagConstraints.NORTHWEST
+            weightx = 1.0
+            insets = Insets(5, 5, 5, 5)
+        }
+
+        val DESCRIPTION_FONT = Font("Arial", Font.PLAIN, PLAN_CONSTRAINTS.STEP_DESCRIPTION_FONT_SIZE)
```

Comment:
Have we confirmed about this font with UX?

Reply:
Yes